lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alkemio/notifications-lib",
-  "version": "0.2.1",
+  "version": "0.2.5",
```

Comment:
Please link your packages locally and don't push multiple versions for one release. It makes no sense to go from `0.2.1` to `0.2.5` in one release...